client/maps/map-preview.tsx

```diff
@@ -7,7 +7,9 @@ import { background400 } from '../styles/colors'
 import { batchGetMapInfo } from './action-creators'
 import MapImage from './map-image'
 
-const StyledDialog = styled(Dialog)`
+const StyledDialog = styled(Dialog)<{ $mapWidth?: number; $mapHeight?: number }>`
+  --sb-map-width: ${props => props.$mapWidth};
```

Comment:
This is not an equivalent change. When we use CSS variables in `style` attributes it's to prevent re-rendering the style every time they change. If you put them into the CSS in this way, you've reverted to adding a new class to the body for every new value, and thus CSS variables are not very useful. Please revert

Reply:
Yeah, I thought about that. But the problem I have now is that we need to apply `styles` generated by the `react-spring` to the dialog as well. I guess it should be possible to combine those styles with these ones, but I was too lazy to figure it out :d. I'll take a second look.

Reply:
It's just an object, you can spread them both into a single object